src/test/java/org/springframework/cloud/deployer/KubernetesTestSupport.java

```diff
@@ -14,20 +14,22 @@
  * limitations under the License.
  */
 
-package org.springframework.cloud.deployer.spi.kubernetes;
+package org.springframework.cloud.deployer;
```

Comment:
any particular need for the package change here? the test check is still for "kubernetes" rather than something generic. if reverted, the protected->public changes below can go too

Reply:
The idea was to use `KubernetesTestSupport` in both the deployer and scheduler tests.